test/client_bandit_test.py

```diff
@@ -100,7 +101,7 @@ def test_get_bandit_action_bandit_does_not_exist():
 def test_get_bandit_action_flag_without_bandit():
     client = get_instance()
     result = client.get_bandit_action(
-        "a_flag", "subject_key", DEFAULT_SUBJECT_ATTRIBUTES, [], "default_variation"
+        "a_flag", "subject_key", DEFAULT_SUBJECT_ATTRIBUTES, {}, "default_variation"
```

Comment:
Did not catch that this test was broken because `is_graceful_mode` was set to True